src/algebra/order/smul.lean

```diff
@@ -142,10 +142,36 @@ lemma bdd_above.smul_of_nonneg (hs : bdd_above s) (hc : 0 ≤ c) : bdd_above (c
 
 end ordered_smul
 
+/-- To prove that a linear ordered module is ordered, it suffices to verify only the first axiom of
```

Comment:
```suggestion
/-- To prove that a linear ordered monoid is an ordered module, it suffices to verify only the first axiom of
```
something like this?

Reply:
Yeah that's fair. My docstring was confusing.